### PR TITLE
Fix dropship related crash

### LINKED ISSUE #588: Dropship related crash
Sent to me by Iniquity

```
[08:09:27] [SCRIPT SV] [info] SCRIPT ERROR: [SERVER] Player is already alive.
[08:09:27] [SCRIPT SV] [info]  -> this.Code_RespawnPlayer( ent )
[08:09:27] [SCRIPT SV] [info] 
CALLSTACK
*FUNCTION [CPlayer()] class/CPlayer.nut line [72]
*FUNCTION [SpawnPlayerIntoDropship()] mp/_classic_mp_dropship_intro.gnut line [191]

[08:09:27] [SCRIPT SV] [info] LOCALS
[ent] NULL
[this] ENTITY (player killerofkingsYT [1] (player "killerofkingsYT" at <-3785.33 -1505.03 715.668>))
[playerDropshipIndex] 1
[playerDropship] unknown struct
[teamDropships] ARRAY
[player] ENTITY (player killerofkingsYT [1] (player "killerofkingsYT" at <-3785.33 -1505.03 715.668>))
[this] TABLE

DIAGPRINTS
```

### PR DESCRIPTION
**Pull request by EnderBoy9217**
Adds a failsafe to fix the bug identified in #588.

ikhadhonger and CTalvio have already been running this code as a mod for over a month

Closes #588

### CODE REVIEW

**Comment by EnderBoy9217** [comment]
Server with has run with this for 13 hours without error, previously it had crashed 4 times within 12 hours.

**Comment by ASpoonPlaysGames** [comment]
How? Aren't asserts disabled in our build?

**Comment by EnderBoy9217** [comment]
Should I rewrite it just because we don't have a proper way to test?

**Comment by ASpoonPlaysGames** [comment]
Yeah, but dont do it in cplayer, the original issue is coming from `SpawnPlayerIntoDropship()` in `mp/_classic_mp_dropship_intro.gnut` line 191
Add a check there, not here

**Comment by EnderBoy9217** [comment]
Changed the fix to what @ASpoonPlaysGames
This turned out to be the same fix ikhadhonger also used to fix their servers

**Comment by EnderBoy9217** [comment]
one thing I will note is these couple of lines before
```
if ( IsAlive( player ) )
		player.Die() // kill them so we don't have any issues respawning them later
```

File: Northstar.CustomServers/mod/scripts/vscripts/mp/_classic_mp_dropship_intro.gnut [hunk 1]
```diff
@@ -255,4 +256,4 @@ void function PlayerJumpsFromDropship( entity player )
 		WaitFrame()
 	
 	TryGameModeAnnouncement( player )
-}
\ No newline at end of file
+}
```
Comment (F1F7Y) [inline Northstar.CustomServers/mod/scripts/vscripts/mp/_classic_mp_dropship_intro.gnut:259]: Unnecessary whitespace change, otherwise looks good

**Review by F1F7Y — CHANGES_REQUESTED** [review]

File: Northstar.CustomServers/mod/scripts/vscripts/mp/_classic_mp_dropship_intro.gnut (hunk 1 above)
Comment (ASpoonPlaysGames) [reply Northstar.CustomServers/mod/scripts/vscripts/mp/_classic_mp_dropship_intro.gnut:259]: IMO this whitespace change is fine, files should have an ending newline rly

**Review by F1F7Y — APPROVED** [review]
Pinged mentioned parties in discord to verify this, code looks good
